crate/utils/src/crypto/curve_25519/operation.rs

```diff
@@ -192,87 +203,20 @@ fn create_p_curve_key_pair(
     Ok(KeyPair::new(private_key, public_key))
 }
 
-/// Generate a P-192 Key Pair. Not FIPS-140-3 compliant. **This curve is for
```

Comment:
Maybe we could keep those comments. They are a good indicator of where to look in the standards.

Reply:
Those comments have been moved to implementation.rs